variables.tf

```diff
@@ -129,12 +129,6 @@ variable "config_accounts" {
   type        = "list"
 }
 
-variable "alb_accounts" {
-  description = "List of accounts for ALB logs.  By default limits to the current account."
-  default     = []
-  type        = "list"
-}
-
```

Comment:
The format `"arn:aws:s3:::%s/%s/AWSLogs/%s/*"` as previously used with multiple ALBs was wrong.  Since each ALB will typically log its own name, e.g, `/alb/<ALB_NAME>/AWSLogs/%s/<ACCOUNT ID>`, so iterating on multiple account ids wasn't really what we wanted.  By switching to one list of ALB prefixes, we can properly handle complex deployments (as shown in the example).

Reply:
So, the concern I have here is that you're also going to change to location of where those logs are going, which seems like a breaking change. I am not sure how this will affect folks who are using the current version of the module with regard to their alerting, etc. At the very least, this should mean a minor version change and a note in the README. I'd like to see @chrisgilmerproj weigh in here too if he's using this on his projects.

Reply:
I kind of feel that the current version is probably broken (and likely no one is actually using it).

Reply:
`arn:aws:s3:::my-log-bucket/my-log-prefix/AWSLogs/1234566789/*` and `arn:aws:s3:::my-log-bucket/my-log-prefix/*` seem different to me. I know *some* people are using the module -- do you just mean that they aren't using this account ID parameter?

I guess your point is that this is only use for resource permissions and the new expression is just more inclusive, so it shouldn't be a breaking change in that way? That makes sense to me I think.

Reply:
Yeah, I don't think anyone is using `alb_accounts`.  It's not a "breaking" change per-se, but would require users to update to use the `alb_logs_prefixes` variable.